src/main/java/scala/compat/java8/wrappers/IteratorPrimitiveDoubleWrapper.java

```diff
@@ -0,0 +1,13 @@
+package scala.compat.java8.wrappers;
+
+public class IteratorPrimitiveDoubleWrapper implements java.util.PrimitiveIterator.OfDouble {
```

Comment:
I left them non-final since that seemed to be idiomatic Java.  (The JIT compiler is really good at detecting monomorphic access.)  But the implementation is trivial; it wouldn't hurt to make them final either.  Might be a good signal that this just a particular implementation that we happen to be using.